src/Kentico.Xperience.Lucene/Admin/Client/src/templates/LuceneIndexConfigurationFormComponent.tsx

```diff
@@ -1,309 +1,430 @@
-      }
-    } else {
-      const rowIndex = rows.findIndex((x) => {
-        return x.identifier === editedIdentifier;
-      });
-
-      if (rowIndex === -1) {
-        alert('Invalid edit');
-      }
-
-      const newRows = rows;
-      const editedRow = rows[rowIndex];
-      const pathCellInNewRow = rows[rowIndex].cells[0] as StringCell;
-      pathCellInNewRow.value = path;
-      const propPathIndex = props.value.findIndex(
-        (p) => p.aliasPath === editedIdentifier,
-      );
-      const updatedPath: IncludedPath = {
-        aliasPath: path,
-        identifier: props.value[propPathIndex].identifier,
-        contentTypes: contentTypesSplit,
-      };
-      props.value[propPathIndex] = updatedPath;
-
-      editedRow.cells[0] = pathCellInNewRow;
-      editedRow.identifier = path;
-
-      newRows[rowIndex] = editedRow;
-      setRows(newRows);
     }
 
-    setEditedIdentifier('');
-    setShowPathEdit(false);
-    setShowAddNewPath(true);
-  };
-  const addNewPath = (): void => {
-    setShowPathEdit(true);
-    setContentTypesValue('');
-    setPath('');
-    setEditedIdentifier('');
-    setShowAddNewPath(false);
-  };
-
-  return (
-    <Stack>
-      <Table
-        columns={prepareColumns()}
-        rows={rows}
-        onRowClick={showContentItems}
-      />
-      {showPathEdit && (
-        <div>
-          <br></br>
-          <Input
-            label="Included Path"
-            value={path}
-            onChange={handleInputChange}
-          />
-          <br></br>
-          <div className="label-wrapper___AcszK">
-            <label className="label___WET63">Available content types:</label>
-          </div>
-          <ul>
-            {props.possibleItems.map((x) => (
-              <label key={x} className="label___WET63">
-                <li key={x}>
-                  <span>{x}</span>
-                </li>
-              </label>
-            ))}
-          </ul>
-          <br></br>
-          <TextArea
-            label="Included ContentType Items"
-            value={contentTypesValue}
-            onChange={handleTextareaChange}
-          />
-          <br></br>
-          <div className="label-wrapper___AcszK">
-            <label className="label___WET63">
-              <span>Separate each content type with a new line</span>
-            </label>
-          </div>
-          <br></br>
-          <Button
-            type={ButtonType.Button}
-            label="Save Path"
-            onClick={savePath}
-          ></Button>
-        </div>
-      )}
-      <br></br>
-      {showAddNewPath && (
-        <Button
-          type={ButtonType.Button}
-          label="Add new path"
-          onClick={addNewPath}
-        ></Button>
-      )}
-    </Stack>
-  );
+    const customStyle: StylesConfig<OptionType, true, GroupBase<OptionType>> = {
+        control: (styles, { isFocused }) => ({ // eslint-disable-line @typescript-eslint/consistent-type-assertions
```

Comment:
In this case it would make a sense to disable `consistent-type-assertions` for the whole customStyle block with
```
/* eslint-disable @typescript-eslint/naming-convention */
..... block code ....
/* eslint-enable @typescript-eslint/naming-convention */ 
```